server.js

```diff
@@ -1,5 +1,16 @@
-const app = require("./app");
+const mongoose = require("mongoose");
 
-app.listen(3000, () => {
-  console.log("Server running. Use our API on port: 3000");
+const uri = "mongodb+srv://wiola:Janczura99@cluster0.cyhix7v.mongodb.net/test";
```

Comment:
To powinno być przeniesione do pliku .env
DB_PATH=mongodb+srv://<login>:<pass>@cluster0.cyhix7v.mongodb.net/<db>
i odczytywane w ten sposób
const uri = process.env.MONGODB_URI

A teraz masz wyciek hasła i jeśli dane były by prawdziwe to problem z GDPR.